src/systemsSerology.jl

```diff
@@ -1,40 +1,75 @@
 using DataFrames
 using CSV
 
+""" Subset systems serology dataset for HIV1.p66 """
+function HIV1p66sub()
+    df = FcgR.importAlterMSG()
+
+    # Find all rows that contain data for HIV1.p66
+    #create a subsetted dataframe for HIV1.p66 data
+    return df[occursin.("HIV1.p66", df.Sig), :]
+end   
+
 
 """ Plot HIV1.p66 data in terms of FcgRIIIa vs. ADCC data"""
 function plotHIV1p66()
     dataDir = joinpath(dirname(pathof(FcgR)), "..", "data")
     dfF = CSV.read(joinpath(dataDir, "alter-MSB", "data-function.csv"))
-
-    df = FcgR.importAlterMSG()
-    dfL = df[occursin.("HIV1.p66", df.Sig), :]
-
+    dfL = HIV1p66sub()
+    
     # Gather all of the ADCC data from "data-function.csv"
-    newdfF = dfF[dfF[!, :ADCC] .!= "NA", :]
-    ADCClist = [parse(Float64, x) for x in newdfF[!, :ADCC]] # All ADCC data in one array of ints
+    newdfF = dfF[dfF[!, :ADCC].!= "NA", :]
+    ADCClist = [parse(Float64,x) for x in newdfF[!, :ADCC]] # All ADCC data in one array of ints
```

Comment:
As a last step before merging, @murphymadeleine21, could  you make this plotting function use your table one? It seems like the data wrangling here is redundant now that you have the cleaner `antigenTables()`.

Reply:
Just pushed the changes.